src/components/Settings/Settings.react.js

```diff
@@ -1,36 +1,455 @@
 // Packages
 import React, { Component } from 'react';
-import RaisedButton from 'material-ui/RaisedButton'
-import PropTypes from 'prop-types'
+import RaisedButton from 'material-ui/RaisedButton';
+import PropTypes from 'prop-types';
+import Cookies from 'universal-cookie';
+import $ from 'jquery';
 
 // Components
+import TextField from 'material-ui/TextField';
 import StaticAppBar from '../StaticAppBar/StaticAppBar';
+import Paper from 'material-ui/Paper';
+import Footer from '../Footer/Footer.react.js'
+import './Settings.css';
+import TimezonePicker from 'react-timezone';
+import List from 'material-ui/List/List';
+import ListItem from 'material-ui/List/ListItem';
+import Divider from 'material-ui/Divider';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import countryData from 'country-data';
+import MenuItem from 'material-ui/MenuItem';
 
+const cookies = new Cookies();
+const token = cookies.get('loggedIn');
+const url = 'https://api.susi.ai/aaa/listUserSettings.json?access_token=' + token;
 
 class Settings extends Component {
 
+	constructor(props) {
+		super(props);
+		this.state = {
+			dataFetched: false,
+			selectedSetting: 'Account',
+			PrefLanguage: 'en-US',
+			TimeZone: '',
+			CountryCode: '',
+			CountryDialCode: '',
+			PhoneNo: '',
+			Server: '',
+			ServerUrl: '',
+			deviceData: false,
+			voiceList: [
+			{
+				lang: 'am-AM',
+				name: 'Armenian'
+			},{
+				lang: 'zh-CH',
+				name: 'Chinese'
+			},{
+				lang: 'de-DE',
+				name: 'Deutsch'
+			},{
+				lang: 'gr-GR',
+				name: 'Greek'
+			},{
+				lang: 'hi-IN',
+				name: 'Hindi'
+			},{
+				lang: 'pb-IN',
+				name: 'Punjabi'
+			},{
+				lang: 'np-NP',
+				name: 'Nepali'
+			},{
+				lang: 'ru-RU',
+				name: 'Russian'
+			},{
+				lang: 'es-SP',
+				name: 'Spanish'
+			},{
+				lang: 'fr-FR',
+				name: 'French'
+			},{
+				lang: 'jp-JP',
+				name: 'Japanese'
+			},{
+				lang: 'nl-NL',
+				name: 'Dutch'
+			},{lang: 'en-US',
+				name: 'US English'
+			}]
+
+		};
+	}
+
+	apiCall = () => {
+		$.ajax({
+			url: url,
+			type:'GET',
+			dataType: 'jsonp',
+			statusCode: {
+				404: function(xhr) {
+					if(window.console) {console.log(xhr.responseText);}
```

Comment:
Can you please explain about the use case of this log?

Reply:
Whenever the user settings are not applicable for a user role or the settings are not found on the accounting model then this error occurs. This is implemented the same way as on chat.susi settings. Thanks.